### PR TITLE
Adds istanbul ignore to Asset Saver

### PR DESCRIPTION
**Pull request by paullewis**
TBR @paulirish

### CODE REVIEW

**Comment by paulirish** [comment]
lgtm

**Comment by paulirish** [comment]
travis error is this:

image

**Comment by brendankenny** [comment]
it's #403 (introduced in #400)

**Comment by paulirish** [comment]
fixing now.

**Comment by paulirish** [comment]
failure fixed by ec140b93670c96fcd27563fa5dbc7f1e77816724

File: src/lib/asset-saver.js [hunk 1]
```diff
@@ -72,6 +72,8 @@ img {
   `;
 }
 
+// Set to ignore because testing it would imply testing fs, which isn't strictly necessary.
+/* istanbul ignore next */
```
Comment (brendankenny) [inline src/lib/asset-saver.js]: wait, why is this being ignored? It seems even easier to check than [`saveAssets`](https://github.com/GoogleChrome/lighthouse/blob/b1e091957e045a38b71d077eb6b161cfd394c02a/test/src/lib/asset-saver.js#L51-L63)

**Pull request merged**